src/content/getting-started/create-workload-cluster/index.md

```diff
@@ -107,6 +110,24 @@ kubectl gs template cluster \
   > cluster.yaml
 ```
 
+{{< /tab >}}
+{{< tab id="cluster-capvcd" for-impl="capvcd">}}
+
+The VMware Cloud Director provider is not yet supported by `kubectl gs template cluster` but you can use the [example manifest](https://github.com/giantswarm/cluster-cloud-director/tree/main/examples) provided in the cluster chart's repo.
+
+Make sure to replace the relevant fields to fit your own VCD environment.
+
+This will install the relevant Helm charts [cluster-cloud-director](https://github.com/giantswarm/cluster-cloud-director) and [default-apps-cloud-director](https://github.com/giantswarm/default-apps-cloud-director) (bundle of default apps):
```

Comment:
```suggestion
This will install the relevant Helm charts [cluster-cloud-director](https://github.com/giantswarm/cluster-cloud-director) and [default-apps-cloud-director](https://github.com/giantswarm/default-apps-cloud-director) (bundle of default apps).
```